src/legacyCompat.ts

```diff
@@ -145,9 +145,12 @@ function parseLegacyWmsGetMapParams(wmsParams: Record<string, any>): ParsedLegac
     getMapParams.height = height;
   }
 
-  if (params.maxcc) {
-    getMapParams.maxCCPercent = parseInt(params.maxcc);
-  }
+  // Instead of dealing with maxCC at this point, we pass it through as an unknown parameter. In the
+  // future, it should be used to instantiate the layers instead.
+  // if (params.maxcc) {
+  //   getMapParams.maxCloudCoverPercent = parseInt(params.maxcc);
+  // }
```

Comment:
This could probably be deleted right before merging when it's all ready.

Reply:
Actually I left it here on purpose, as a message - it is a reminder that this is not done properly, because `maxcc` is something that we _do_ know how to handle, so passing it through `unknown` params is not correct. Still, without the named parameters in constructors this wolud be difficult to do - so I would like to leave it for later (when the constructors are changed) if you agree?

Reply:
Yeah of course, let's leave it. Resolving.